frontend/src/metabase/questions/containers/CollectionEditorForm.jsx

```diff
@@ -41,12 +42,14 @@ export default class CollectionEditorForm extends Component {
                 form
                 title={fields.id.value != null ? fields.name.value : "New collection"}
                 footer={[
-                    <Button className="mr1" onClick={onClose}>
-                        Cancel
-                    </Button>,
-                    <Button primary disabled={invalid} onClick={handleSubmit}>
-                        { fields.id.value != null ? "Update" : "Create" }
-                    </Button>
+                    <div ref={(ref) => { this.footerRef = ref}}>
```

Comment:
The alternative is to move the content and `title` and `footer` props to a separate component that uses `ModalContent` directly, then test that instead of the component that returns a `Modal`.

I'm wondering if `Modal` should actually just be a higher order component anyway.